resources/tentative/todomvc/src/components/todo-app/todo-app.component.js

```diff
@@ -0,0 +1,171 @@
+import template from "./todo-app.template.js";
+import TodoTopbar from "../todo-topbar/todo-topbar.component.js";
+import TodoList from "../todo-list/todo-list.component.js";
+import TodoBottombar from "../todo-bottombar/todo-bottombar.component.js";
+import { useRouter } from "../../hooks/useRouter.js";
+
+// import { data } from "../../data.js";
+
+class TodoApp extends HTMLElement {
+    constructor() {
+        super();
+
+        // elements
+        this.topbar = undefined;
+        this.list = undefined;
+        this.bottombar = undefined;
+        // state
+        this.connected = false;
```

Comment:
This boolean seems unnecessary given we have https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected